src/lightning/fabric/cli.py

```diff
@@ -44,7 +48,33 @@ def _get_supported_strategies() -> List[str]:
 if _CLICK_AVAILABLE:
     import click
 
-    @click.command(
+    def _legacy_main() -> None:
+        """Legacy CLI handler for fabric.
+
+        Raises deprecation warning and runs through fabric cli if necessary, else runs the entrypoint directly
+
+        """
+        warnings.warn("lightning run model is deprecated and will be removed in future versions."
+            " Please call `fabric run model` instead.", DeprecationWarning)
```

Comment:
Since this is a CLI interaction, we should use print (we use print for errors as well)

Reply:
done